lib/internal/modules/helpers.js

```diff
@@ -51,41 +57,63 @@ function initializeCjsConditions() {
   ]);
 }
 
+/**
+ * Get the conditions that apply to the CommonJS loader.
+ */
 function getCjsConditions() {
   if (cjsConditions === undefined) {
     initializeCjsConditions();
   }
   return cjsConditions;
 }
 
-function loadBuiltinModule(filename, request) {
-  if (!BuiltinModule.canBeRequiredByUsers(filename)) {
+/**
+ * Provide one of Node.js' public modules to user code.
+ * @param {string} id The identifier/specifier of the builtin module to load
```

Comment:
In some of the JSDoc comments you added `-` between the variable name and description, but not in here. Is this correct?

```suggestion
 * @param {string} id - The identifier/specifier of the builtin module to load
```